version/util.go

```diff
@@ -15,25 +15,40 @@ limitations under the License.
 package version
 
 import (
+	"strconv"
 	"strings"
 )
 
 var (
-	validCurrentVersions = map[string]bool{
-		"2.6.0": true, "2.7.0": true, "2.8.0": true, "2.9.0": true,
-		"2.10.0": true, "2.10.1": true, "2.11.0": true,
-	}
+	minCurrentVersion   = "2.6.0"
 	validDesiredVersion = strings.Split(Version, "-")[0]
 )
 
 // IsCurrentVersionValid verifies if the  current version is valid or not
 func IsCurrentVersionValid(v string) bool {
 	currentVersion := strings.Split(v, "-")[0]
-	return validCurrentVersions[currentVersion]
+	return IsCurrentLessThanOrEqualNewVersion(minCurrentVersion, currentVersion) &&
```

Comment:
Probably create another function to make it clear that we are checking for:
- CanCurrentVersionBeUpgraded()
- IsCurrentVersionLessThanOrEqualNewerVersion()

Reply:
Added the function CanCurrentVersionBeUpgraded(currentVersion)

For IsCurrentVersionLessThanOrEqualNewerVersion renamed it to IsOldLessThanOrEqualNewVersion which describes more of the functionality it does. Here I wanted to avoid repeating the for loops so the same function is called with different args to get the range minCurrentversion < version < validDesiredVersion